### PR TITLE
[feature #164832218] Enable the contributor to see the total amount requested

### PR DESCRIPTION
**Pull request by gracelungu**
#### What does this PR do?
Enables the contributor to see the total loans

#### Description of Task to be completed?
- Write the model
- Write the controller

#### How should this be manually tested?
Via Postman

#### What are the relevant pivotal tracker stories?
164832218

### CODE REVIEW

File: models/queries/contributors.js [hunk 1]
```diff
@@ -1,9 +1,11 @@
 const contributorsQueries = {
   contribute:
-    'INSERT INTO contributions (amount, owner) VALUES ($1, $2) RETURNING *',
-  getContrubutions: 'SELECT * FROM contributions WHERE owner=$1',
-  getContrubutionById: 'SELECT * FROM contributions WHERE id=$1',
-  payContribution: 'UPDATE contributions SET status = 1, paiddate = $1 WHERE id = $2 AND status = 0 RETURNING *',
+    "INSERT INTO contributions (amount, owner) VALUES ($1, $2) RETURNING *",
+  getContrubutions: "SELECT * FROM contributions WHERE owner=$1",
+  getLoans: "SELECT * FROM loans ",
```
Comment (Karlmusingo) [inline models/queries/contributors.js]: I think the total amount requested has to be for loans which are pending only, what do you think?

Comment (gracelungu) [reply models/queries/contributors.js]: You are right. let me fix that.

Comment (gracelungu) [reply models/queries/contributors.js]: It has been fixed.

**Review by Karlmusingo — APPROVED** [review]
Well done ...

**Pull request merged**